app/components/shared/__snapshots__/Duration.spec.js.snap

```diff
@@ -60,15 +52,15 @@ exports[`Duration Duration.Full renders as expected 7`] = `
 <span
   className="tabular-numerals"
 >
-  13 days, 16 hours, 22 minutes
+  1 week, 6 days, 16 hours
 </span>
 `;
 
 exports[`Duration Duration.Full renders as expected 8`] = `
 <span
   className="tabular-numerals"
 >
-  8 weeks, 2 days, 20 hours
+  9 weeks, 0 days, 19 hours
```

Comment:
Hmm, that’s annoying that it doesn’t remove zeroed values :(

Reply:
it seems to trim left and right but not middle.

Reply:
I've got a feeling it's being worked toward a 2.0.0 release, you could probably get a feature request or contribution in before then. I'm not quite savvy enough. 😅